email_show_only_changes/models/mail_thread.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, api
+
+
+class MailThread(models.AbstractModel):
+    _inherit = 'mail.thread'
+
+    @api.multi
+    def _message_track(self, tracked_fields, initial):
+        """For a given record, fields to check (column name, column info)
+        and initial values, return a structure that is a tuple containing :
+
+        - a set of updated column names
+        - a list of changes (old value, new value, column name, column info)
+        """
+        self.ensure_one()
+        changes = super(MailThread, self)._message_track(
```

Comment:
Is possible a IndexError here?

Reply:
No, it not possible because _messages_track always return a tuple